### PR TITLE
sql-migration: update migration status page latest design

### PR DESCRIPTION
**Pull request by brian-harris**
This PR adds:
* context menu actions to migration status page
* database icon next to database name
* migration status and error images around migration status
* localize status display text
* add Azure account connection and authentication validation to account selection page

Migration status with context menu
Dark theme
image

Light theme:
image

In progress:
image

### CODE REVIEW

File: extensions/sql-migration/src/dialog/migrationStatus/migrationStatusDialog.ts [hunk 1]
```diff
@@ -152,122 +161,290 @@ export class MigrationStatusDialog {
...
+		}
+		if (properties.migrationStatusDetails?.restoreBlockingReason) {
+			warningCount++;
+		}
+
+		return this._getStatusControl(migrationStatus, warningCount);
+	}
+
+	private _getStatusControl(status: string, count: number): azdata.FlexContainer {
+		status = 'InProgress';
```
Comment (brian-harris) [inline extensions/sql-migration/src/dialog/migrationStatus/migrationStatusDialog.ts:395]: remove, test code.

File: extensions/sql-migration/src/dialog/migrationStatus/migrationStatusDialog.ts [hunk 2]
```diff
@@ -348,21 +525,58 @@ export class MigrationStatusDialog {
...
+					isReadOnly: true,
+					rowCssStyles: rowCssStyle,
+					headerCssStyles: headerCssStyles,
 				}
 			]
 		}).component();
 		return this._statusTable;
 	}
+
+	private _statusImageMap(status: string): azdata.IconPath | undefined {
```
Comment (Charles-Gagnon) [inline extensions/sql-migration/src/dialog/migrationStatus/migrationStatusDialog.ts:553]: You always return an IconPath here, never undefined

File: extensions/sql-migration/src/dialog/migrationStatus/migrationStatusDialog.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (brian-harris) [reply extensions/sql-migration/src/dialog/migrationStatus/migrationStatusDialog.ts]: it was test code removed in 3rd iteration

File: extensions/sql-migration/src/dialog/migrationStatus/migrationStatusDialog.ts [hunk 3]
```diff
@@ -152,122 +161,290 @@ export class MigrationStatusDialog {
...
-				const databaseHyperLink = this._view.modelBuilder.hyperlink().withProps({
-					label: migration.migrationContext.properties.sourceDatabaseName,
-					url: ''
-				}).component();
-				databaseHyperLink.onDidClick(async (e) => {
-					await (new MigrationCutoverDialog(migration)).initialize();
-				});
-				migrationRow.push({
-					value: databaseHyperLink,
+			vscode.commands.registerCommand(
```
[the remark this answers is not in the document]
Comment (brian-harris) [reply extensions/sql-migration/src/dialog/migrationStatus/migrationStatusDialog.ts]: good catch, thank you

File: extensions/sql-migration/src/constants/strings.ts [hunk 4]
```diff
@@ -77,6 +77,9 @@ export const AZURE_TENANT = localize('sql.migration.azure.tenant', "Azure AD ten
 export function ACCOUNT_STALE_ERROR(account: AzureAccount) {
 	return localize('azure.accounts.accountStaleError', "The access token for selected account '{0}' is no longer valid. Please click the 'Link Account' button and refresh the account or select a different account.", `${account.displayInfo.displayName} (${account.displayInfo.userId})`);
 }
+export function ACCOUNT_ACCESS_ERROR(account: AzureAccount, error: Error) {
+	return localize('azure.accounts.accountAccessError', "Am error occurred while accessing the selected account '{0}'. Please click the 'Link Account' button and refresh the account or select a different account. Error '{1}'", `${account.displayInfo.displayName} (${account.displayInfo.userId})`, error.message);
```
Comment (aasimkhan30) [inline extensions/sql-migration/src/constants/strings.ts:81]: Nit: An

**Review by aasimkhan30 — APPROVED** [review]

**Pull request merged**